addons/mass_mailing/models/mailing_list.py

```diff
@@ -126,6 +126,33 @@ def copy(self, default=None):
     # ACTIONS
     # ------------------------------------------------------
 
+    def action_open_import(self):
+        """Open the mailing list contact import wizard."""
+        self.ensure_one()
+        action = self.env['ir.actions.actions']._for_xml_id('mass_mailing.mailing_contact_import_action')
+        action['context'] = {
+            'default_mailing_list_ids': self.ids,
+            'default_subscription_list_ids': [(0, 0, {'list_id': self.id})],
+        }
+        return action
+
+    def action_send_mailing(self):
+        """Open the mailing form view, with the current lists set as recipients."""
```

Comment:
Can't we use "mailing_mailing_action_mail" here instead?

Reply:
yes :) I just need to keep the view id